### PR TITLE
feat: Allow to share Mentor Profile

### PR DESCRIPTION
**Pull request by Deejarh**
## Description

-  A feature to allow users copy/share mentor.

## Change Type
- [ ] Bug Fix
- [x] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Data Update
- [ ] Documentation
- [ ] Other

## Screenshots

**Before**
Screenshot 2024-07-10 at 10 36 41

**After**
Screenshot 2024-07-10 at 12 00 54

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _includes/share-mentor.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{% assign base_url = 'https://womencodingcommunity.com' %}
```
Comment (dricazenck) [inline _includes/share-mentor.html:1]: You could create `base_url` in _config.yml and re-use like this `site.base_url`

File: _includes/share-mentor.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            href="mailto:?subject=Check out this mentor&body=Check out this mentor: {{ mentor.name }} at {{ full_url }}"
+            target="_blank">
+            <span class="share-icon"> {% include icons/envelope.svg %} </span>
+            <span class="ml-3">Email a friend</span>
+        </a>
+    </li>
+</ul>
+
+<script>
+  function copyToClipboard(text) {
```
Comment (dricazenck) [inline _includes/share-mentor.html]: javascript/jquery should be in Js files, for example [here](https://github.com/Women-Coding-Community/WomenCodingCommunity.github.io/blob/main/assets/js/mentors.js)

File: _includes/share-mentor.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    navigator.clipboard.writeText(text).then(
+      function () {
+        alert("Link copied");
```
Comment (dricazenck) [inline _includes/share-mentor.html]: Please use bootstrap notification, For example [this](https://getbootstrap.com/docs/4.0/components/alerts/), you can hide and show as we do with other components:

image

There are examples already [here](https://github.com/Women-Coding-Community/WomenCodingCommunity.github.io/blob/main/assets/js/mentors.js#L18-L20)

**Comment by dricazenck** [comment]
Thank you for the collaboration!! just small details but I loved your initiative!

File: _includes/share-mentor.html (hunk 1 above)
Comment (Deejarh) [reply _includes/share-mentor.html:1]: @dricazenck  done

File: _includes/share-mentor.html (hunk 2 above)
Comment (Deejarh) [reply _includes/share-mentor.html]: @dricazenck  done, i also updated the javascript functionality to jquery for consistency .

File: _includes/share-mentor.html (hunk 3 above)
Comment (Deejarh) [reply _includes/share-mentor.html]: @dricazenck  
Screenshot 2024-07-10 at 16 40 26

File: assets/js/mentors.js [hunk 4]
```diff
@@ -136,6 +136,20 @@ var controllerMentors = (function(jQuery) {
...
 
+    jQuery("#copy-link").click(function () {
+      var text = jQuery(this).data("url");
+      var tempInput = jQuery("<input>");
+      jQuery("body").append(tempInput);
+      tempInput.val(text).select();
+      document.execCommand("copy");
+      tempInput.remove();
+
+      jQuery("#link-msg").removeClass("d-none");
```
Comment (dricazenck) [inline assets/js/mentors.js]:
```suggestion
      jQuery("#link-msg").removeClass(CLASS_HIDDEN);
```

File: assets/js/mentors.js [hunk 5]
```diff
@@ -136,6 +136,20 @@ var controllerMentors = (function(jQuery) {
...
+      setTimeout(function () {
+        jQuery("#link-msg").addClass("d-none");
```
Comment (dricazenck) [inline assets/js/mentors.js]:
```suggestion
        jQuery("#link-msg").addClass(CLASS_HIDDEN);
```

File: assets/js/mentors.js [hunk 6]
```diff
@@ -136,6 +136,20 @@ var controllerMentors = (function(jQuery) {
         });
     };
 
+    jQuery("#copy-link").click(function () {
```
Comment (dricazenck) [inline assets/js/mentors.js:139]:
```suggestion
    jQuery("#copy-link").click(function () {
```

Comment (dricazenck) [inline assets/js/mentors.js]: check the code on this page, you should follow the current style of implementing. Both ways works, but would be better to keep like that: 

``` 
var copyLink = jQuery('#link-msg');

copyLinkt.click(function() {
     ... 
} 
```

File: assets/js/mentors.js [hunk 7]
```diff
@@ -136,6 +136,20 @@ var controllerMentors = (function(jQuery) {
         });
```
Comment (dricazenck) [inline assets/js/mentors.js:139]: move your logic here, for example.

File: assets/js/mentors.js (hunk 6 above)
Comment (Deejarh) [reply assets/js/mentors.js]: Thank you for pointing that out. I'll make sure to follow the current style in the future. Apologies for the oversight! 🙃

**Comment by Deejarh** [comment]
@dricazenck  I'm currently using `{% assign base_url = site.base_url %}`, but it's not picking up the base URL from `_config.yml`. It was working before my last push, so I'm unable to pinpoint the issue. I've already tried restarting my server. Should I add the URL directly instead? What do you think?

**Review by dricazenck — APPROVED** [review]

**Pull request merged**